REVISED_README.md

```diff
@@ -0,0 +1,72 @@
+# Fitness-Accountability
+<a href="https://t.co/DOcdsUEQmP ">Join our Discord community here!</a><br>
+<a href="https://fitdevs-withkat.github.io/Support/landing_page/">Quick Introduction</a>
+
+### Quick access index
+
+[Step 1 - Creating a JSON file in the data directory](#step-1---creating-a-json-file-in-the-data-directory)<br>
+[Step 2 - Inputting information in the JSON file](#step-2---inputting-information-in-the-json-file)<br>
+[Step 3 - Commit the file](#step-3---commit-the-file)<br>
+[Step 4 - Create a pull request](#step-4---create-a-pull-request)<br>
+[]()<br>
```

Comment:
This can be removed?

Reply:
I was thinking it would be a mini contents section but if its not needed, i will remove it